src/Plugin.php

```diff
@@ -55,6 +73,9 @@ public function __construct(array $config = [])
             throw new \InvalidArgumentException("Missing required configuration key 'nicks'");
         }
         $this->setNicks($config['nicks']);
+        if (!empty($config['recovery'])) {
```

Comment:
How so?

    A variable is considered empty if it does not exist or if its value equals FALSE.

So if `$config['recovery']` is set to `false`, then `empty($config['recovery'])` evaluates to `true`, and the `!` operator inverts it to `false`.

Suppose it doesn't really matter. The difference in behavior is that `$this->recovery` will never be set (i.e. will be `null`) if `$config['recovery']` is `false` and thus will still evaluate to being false-y when used as a boolean value.